deploy/Chart/templates/ucp/secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.global.caCert.filename }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.global.caCert.secretName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: controller
+    app.kubernetes.io/part-of: radius
+data:
+  ca.crt: {{ .Files.Get .Values.caCert.filename | b64enc }}
+{{- end}}
```

Comment:
Is this secret used by all of our deployments? I'd suggest moving it out of the UCP folder.

Reply:
ok let me find the better place.

Reply:
moved it to global directory